library/common/extensions/cert_validator/platform_bridge/platform_bridge_cert_validator.h

```diff
@@ -83,26 +83,21 @@ class PlatformBridgeCertValidator : public CertValidator, Logger::Loggable<Logge
                                     uint8_t tls_alert, OptRef<Stats::Counter> error_counter);
 
   private:
-    Event::SchedulableCallbackPtr next_iteration_callback_;
     PlatformBridgeCertValidator& parent_;
-    std::vector<envoy_data> certs_;
-    std::string host_name_;
+    const std::vector<envoy_data> certs_;
+    const std::string hostname_;
+    const std::vector<std::string> subject_alt_names_;
     Ssl::ValidateResultCallbackPtr result_callback_;
-    const Network::TransportSocketOptionsConstSharedPtr transport_socket_options_;
   };
 
-  // Calls into platform APIs in a stand-alone thread to verify the given certs.
-  // Once the validation is done, the result will be posted back to the current
-  // thread to trigger callback and update verify stats.
```

Comment:
Can you move this comment to PengValidation::verifyCertsByPlatform()?

Reply:
Done!